skmob/tessellation/tilers.py

```diff
@@ -274,7 +267,6 @@ def _build(self, base_shape, meters, crs=constants.DEFAULT_CRS):
 
         # We work with the universal crs epsg:3857
         tmp_crs = constants.DEFAULT_CRS
```

Comment:
The value assigned to this variable is never. Can you please remove it?

Reply:
fix in 558b121